src/Lecture2.hs

```diff
@@ -39,9 +41,11 @@ module Lecture2
     , eval
     , constantFolding
     ) where
+import Data.Monoid (Any)
 
--- VVV If you need to import libraries, do it after this line ... VVV
 
+-- VVV If you need to import libraries, do it after this line ... VVV
+{-# LANGUAGE NoMonomorphismRestriction #-}
```

Comment:
I'm not sure it will disable the extension if you write it here (and it shouldn't be disabled). Will the compiler output change if you remove this line?